Formula/php.rb

```diff
@@ -33,27 +33,33 @@ class Php < Formula
   depends_on "gmp"
   depends_on "icu4c"
   depends_on "jpeg"
+  depends_on "libiconv" if MacOS.sdk_path_if_needed
```

Comment:
It's possible we want to use `if DevelopmentTools.clang_build_version >= 1000` here; I'm not sure how "kosher" it is to use `MacOS.sdk_path_if_needed` in this context.

Reply:
Ok I'll change that around, we can always adapt later if needed.